### DISCUSSION TITLE
Stuck on THN #18

### DISCUSSION THREAD

**Discussion by JSchilders1966** [thehighnibble/thehighnibble.github.io#92] [IMSAI8080esp replica]
I followed the YouTube videos, but after TNN #18 I get stuck. When I switch on the IMSAI 8080 (power on esp32) with the power switch off, all LEDs turn on. When I pressed reset all leds off. But when i turn on the power switch po. I see via a serial console that the IMASI boots but no LEDs light up. Even when I test the pads S3 with the test cable, no LEDs turn on.

I checked the soldering and tested the LEDs in the previous steps

The ICs are in and i have checked the contacts.

What could be the cause, I've already tried everything

**Comment by GrantMeStrength** [comment]
So you haven't put the switches in yet? I think it sometimes acts weird until the switches are in. I'd keep going.

**Reply by JSchilders1966** [reply]
Thanks for your reply, i solder the Power switch, but same result. 
Do i solder all switches and hope for the best :-) ?

**Reply by JSchilders1966** [reply]
Finally i had some time to continue the build. So far still the same problem. The SD cards starts and if i do the reset is see some activity on de serial monitor but no led are on or off. All led turns on after is give the USB power but when i switches the power button, all led go's off and stays off. Even after is solder all the switches noting is working.  Also where i press de reset, no led go's on or off. 

I checks all the solders and followed the assemble video' s of The High Nibble to the letter.  When is tested alle led without the 74HC595N stiftregisters all the leds are working,

Its very disappointing, can some one help me?

**Comment by nbreeden2** [comment]
Make sure you've soldered the extra 6 pins to the ESP32-PICK-KIT processor.

Make sure you are using a quality USB cable. Try a couple different ones to see if it acts differently. Cheap cables cause all kinds of weird issues.

Make sure the address/date switches (ones to the left and middle) are the toggle ones. They should click up or down. None should return to the center position.

For the six switches on the right the 5 with full paddles should always return to center. Pressing them up or down then releasing them should result in the paddle being back in the middle after you release them.

I don't know which switches you have soldered in, so you may not be able to do these tests. If not don't solder in any switches to be able to do the test.
Hold UP the reset switch, for the IMSAI version all LEDs should light. (On the Cromemco version the 2MHz/4MHz LED won't lite. There is a firmware bug that will be addressed in a later firmware release).
Hold down the RESET switch to the "EXT. CLR" position and keep holding it down. The LEDs that are off should all flash, any already on will stay on.
When you release the RESET switch from the "EXT. CLR" position no LEDs should change.
Press and release "STOP".
Press "EXAMINE NEXT" several times, the "ADDRESS BUS" row of LEDs should count upward starting with the LEDs to the far right. You may see random bits in the "DATA BUS" LEDs, this is normal.

Please take some photos of the build as it currently is. Both front and back of the PCB.

-Neil
